frontend/src/pages/modelServing/screens/global/utils.ts

```diff
@@ -12,18 +13,31 @@ export const getTokenDisplayName = (secret: SecretKind): string =>
 export const getInferenceServiceActiveModelState = (
   is: InferenceServiceKind,
 ): InferenceServiceModelState =>
-  <InferenceServiceModelState | undefined>is.status?.modelStatus?.states?.activeModelState ||
-  <InferenceServiceModelState | undefined>is.status?.modelStatus?.states?.targetModelState ||
+  (is.status &&
```

Comment:
Made more explicit type guarding due to these type-check tests:

```
src/pages/modelServing/screens/global/utils.ts:16:16 - error TS18048: 'is.status.modelStatus' is possibly 'undefined'.

16   asEnumMember(is.status?.modelStatus.states.activeModelState, InferenceServiceModelState) ||
```

Reply:
```
export const getInferenceServiceActiveModelState = (
  is: InferenceServiceKind,
): InferenceServiceModelState =>
  asEnumMember(is.status?.modelStatus?.states?.activeModelState, InferenceServiceModelState) ||
  asEnumMember(is.status?.modelStatus?.states?.targetModelState, InferenceServiceModelState) ||
  InferenceServiceModelState.UNKNOWN;
```